dashboard/static/js/base.js

```diff
@@ -3,25 +3,25 @@ $(document).ready(function(){
 
     // This is the js that powers the search box
     $(':input[name=filter]').on('input', function() {
-        // Get value just typed into textbox -- see .toLowerCase()
+            // Get value just typed into textbox -- see .toLowerCase()
```

Comment:
nit: indent issues

Reply:
I tried making eslint happy in this project twice.  I'll catch the eslint stuff with nemo on the next merge.